Tools/MGCB/CommandLineParser.cs

```diff
@@ -284,6 +284,8 @@ private IEnumerable<string> Preprocess(IEnumerable<string> args)
                 if (arg.StartsWith("/@:") || arg.StartsWith("-@:"))
                 {
                     var file = arg.Substring(3);
+                    Directory.SetCurrentDirectory(Path.GetDirectoryName(file));
```

Comment:
Pipeline sets working directory in: https://github.com/mono/MonoGame/blob/develop/Tools/Pipeline/Common/PipelineController.cs#L475 so it wasn't noticeable with it.

Plus it's not like this is changing much with this, working directory can still be overwritten with "workingDir" argument.

Also I've actually known about this for about 9 months or so... http://community.monogame.net/t/starting-directory-for-mgcb-when-loading-response-file-wrong/1608 I just forgot about it till now....